### PR TITLE
feat: create note

### PR DESCRIPTION
**Pull request by devsurajrai**
Added the create new note feature. Please review the code and provide your valuable comments.

LIVE: https://deploy-preview-6--note-phoenix.netlify.app/

### CODE REVIEW

**Review by saifudinbadani — APPROVED** [review]
File and folder structure looks good. Data persistence for notes is good way to handle data.

**Review by Dev-AkshatGupta — COMMENTED** [review]
In UI once user is logged in or signed in remove the log in sign in buttons

File: src/components/CreateNoteCard/CreateNoteCard.jsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-import { useNotesDataContext } from "../../contexts/notesDataContext";
+// import { useNotesDataContext } from "../../contexts/notesDataContext";
 import "./create-note-card.css";
 import { v4 as uuidv4 } from "uuid";
-import { addNote } from "../../util/addNote/addNote";
 
-export const CreateNoteCard = ({ isCreateNewNote, setIsCreateNewNote }) => {
-  const { setNotesData } = useNotesDataContext();
+import { useAuthContext } from "../../contexts/authContext";
+import { addNote } from "../../util/util";
```
Comment (Dev-AkshatGupta) [inline src/components/CreateNoteCard/CreateNoteCard.jsx:7]: I would suggest you to use absolute imports very easy thing to do

File: src/components/CreateNoteCard/CreateNoteCard.jsx [hunk 2]
```diff
@@ -54,7 +55,7 @@ export const CreateNoteCard = ({ isCreateNewNote, setIsCreateNewNote }) => {
                   <button
                     className="button button-secondary btn-sm m-l-md "
                     onClick={() =>
-                      addNote(setNotesData, setIsCreateNewNote, note, uuidv4)
+                      addNote(note, setIsCreateNewNote, userToken, authDispatch)
                     }
                   >
```
Comment (Dev-AkshatGupta) [inline src/components/CreateNoteCard/CreateNoteCard.jsx:60]: Take care  of your indentation

**Comment by devsurajrai** [comment]
> In UI once user is logged in or signed in remove the log in sign in buttons

yes ! It will be taken care of in later stages

**Pull request merged**